project_task_default_stage/models/project.py

```diff
@@ -0,0 +1,15 @@
+# (c) 2015 Incaser Informatica S.L. - Sergio Teruel
+# (c) 2015 Incaser Informatica S.L. - Carlos Dauden
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from odoo import fields, models
+
+
+class ProjectProject(models.Model):
+    _inherit = 'project.project'
+
+    def _get_type_common(self):
+        ids = self.env['project.task.type'].search([
+            ('case_default', '=', True)])
+        return ids
+
+    type_ids = fields.Many2many(default=_get_type_common)
```

Comment:
Could you please use a lambda there? Then we can inherit from this method.
Also I think that '_get_type_common' could be renamed to '_get_default_type_common' or something similar. but at least prefixed by '_get_default'.

Reply:
I'm not sure what you mean. Did you expect something like this: `default=lambda self: self._get_type_common()`? If yes, then i suggest we just simply put the function name inside quotes: `default='_get_type_common'`. By that, we can inherit from the method also.

Reply:
No, @novawish, default is the only Odoo method that you can't inherit enclosing the method name into quotes. This is because you can pass an string also as default, making Odoo to not know what do you want to make. You have to user the `lambda` trick.

That was for <=v10, but I haven't tried in v11. I suppose it happens the same that in previous versions.

Reply:
@pedrobaeza You are absolutely right. I'm mistaken it with `compute` attribute, fixed with lambda function.